renderer/reducers/network.js

```diff
@@ -159,16 +159,11 @@ export { networkSelectors }
 // ------------------------------------
 // Reducer
 // ------------------------------------
-
 /**
  * networkReducer - Network reducer.
  *
  * @param  {object} state = initialState Initial state
```

Comment:
This whole docblock can probably go too since we aren't defining anything new here.